src/lib/statefulForm.jsx

```diff
@@ -128,9 +135,30 @@ export default function statefulForm(mapPropsToFormConfig) {
             value: target.value
           }
         }
+        if (target.type && target.type === 'email') {
```

Comment:
In fact, same thing here, you can simply add a `pattern` attribute, and use Constraint Validation